src/chartLibraries/dygraph/hoverX.test.js

```
@@ -0,0 +1,532 @@
+
+    hoverX = makeHoverX(mockChartUI)
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe("findClosest functionality", () => {
+    it("returns empty object for non-array points", () => {
+      const event = { offsetX: 100, offsetY: 50 }
+      const result = hoverX.toggle(true)
```

ESLint

[error] 'result' is assigned a value but never used.

(no-unused-vars)